src/server/IoDispatchers.cpp

```diff
@@ -146,6 +146,20 @@ static bool _shouldAttemptHandoff(const Globals& globals,
                                   const CONSOLE_INFORMATION& gci,
                                   CONSOLE_API_CONNECTINFO& cac)
 {
+#ifdef __INSIDE_WINDOWS
+    const bool insideWindows = true;
+#else
+    const bool insideWindows = false;
+#endif
+
+    // If we are outside of Windows, do not attempt a handoff
+    // to another target as handoff is an inbox escape mechanism
+    // to get to this copy!
+    if (!insideWindows)
+    {
+        return false;
+    }

```

Comment:
this is a complicated version of...

```
#ifndef __INSIDE_WINDOWS
return false;
#endif
```

right?

Reply:
Yes but I thought I remembered an annoying unreachable code warning +/- UNREFERENCED_PARAMETER things when I had it the other way. I'll double check.

Reply:
Yep all unreachable code warnings now. I'll def it out and add the unreferenced checks.